docs/recipes/setup_circleci.md

```diff
@@ -0,0 +1,30 @@
+# How to: setup CircleCI to deploy to cloud.gov
+
+The cloud.gov docs recommend using a [space-deployer](https://cloud.gov/docs/services/cloud-gov-service-account/) to deploy applications from continuous integration services such as CircleCI.
```

Comment:
[thought] I wonder if we should say something about how this is already completed, and in what case would you need to revisit.

Reply:
Yes good call!